frappe/hooks.py

```diff
@@ -425,11 +425,21 @@
 	"frappe.utils.telemetry.add_bootinfo",
 ]
 
-get_changelog_feed = [
-	{
-		"title": "Spid",
-		"creation": "2023-04-03 16:56:51.436456",
-		"app_name": "Frappe Framework",
-		"link": "https://frappe.io/wiki",
-	}
-]
+get_changelog_feed = "frappe.hooks.get_feed"
+
+
+def get_feed():
+	return [
+		{
+			"title": "Spid",
+			"creation": "2023-04-03 16:56:51.436456",
+			"app_title": app_title,
```

Comment:
Just a comment for when you implement hooks. We can re-use `app_title` which is already specified in hooks. 

https://github.com/frappe/frappe/blob/85377eddfe332f3e1cbf52a9e175aa9014e5406c/frappe/hooks.py#L4